packages/web/package.json

```diff
@@ -114,30 +115,34 @@
     "@types/is-url": "^1.2.28",
     "@types/node": "^14.14.31",
     "@types/nprogress": "^0.2.0",
-    "@types/react": "^16.9.23",
-    "@types/react-dom": "^16.9.5",
+    "@types/react": "^17.0.57",
+    "@types/react-dom": "^17.0.19",
     "@types/uuid": "^8.3.0",
-    "@typescript-eslint/eslint-plugin": "^2.29.0",
-    "@typescript-eslint/parser": "^2.29.0",
     "cli-progress": "^3.9.0",
     "colors": "^1.4.0",
     "copy-webpack-plugin": "^9.0.1",
     "csv-reader": "^1.0.7",
     "csv-writer": "^1.6.0",
-    "eslint": "^7.32.0",
-    "eslint-config-next": "^11.0.1",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-react": "^7.24.0",
     "express": "^4.17.1",
     "husky": "^4.2.3",
     "jsonc-parser": "^3.0.0",
     "p-retry": "^4.2.0",
     "prettier": "^2.0.1",
     "pretty-quick": "^2.0.1",
-    "prisma": "3.5.0",
+    "prisma": "3.15.2",
```

Comment:
What's the reason for the downgrade? I think we had upgraded to a certain version specifically to get Cascading delete functionality but I haven't gone back and checked exactly what that version was, have you double-checked we don't lose anything we need with the downgrade?

Reply:
The new version is 3.15.2, while the old one was 3.5.0, so that's an upgrade of 10 minor versions right? I think nexus needed something from a newer prisma/prisma-client version, hence the upgrade

Reply:
Oh LOL bloody hell that was dumb sleepy Robin reviewing before bed 🤣 somehow I saw 3.1.5 😂